src/metabase/collections/models/collection.clj

```diff
@@ -338,6 +389,18 @@
                     (tru "A Collection placed in a remote-synced Collection must also be remote-synced."))]
           (throw (ex-info msg {:status-code 400, :errors {:location msg}})))))))
 
+(defenterprise check-allowed-content
+  "Checks contents of a collection before saving it. The OSS implementation is a no-op."
+  metabase-enterprise.library.validation
+  [_model-type _collection-id]
+  true)
```

Comment:
nit if these are boolean functions they should be `check-allowed-content?` since I think these raise in the enterprise version if content is not allowed maybe they could just take the model as an argument and return the model.